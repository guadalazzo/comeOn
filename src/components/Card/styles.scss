.card {
  height: 85%;
  display: flex;
  align-items: center;
  justify-content: space-between;
  flex-direction: column;
  max-width: 80%;
  margin: auto;
  label {
    display: flex;
  }
}