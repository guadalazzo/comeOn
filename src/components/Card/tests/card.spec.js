import React from 'react';
import Card from '../index';
import renderer from 'react-test-renderer';


describe('Test Card component', () => {
    test('Card snapshot', () => {
        const component = renderer.create(<Card>Home</Card>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
