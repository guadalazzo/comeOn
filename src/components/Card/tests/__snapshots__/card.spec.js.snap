// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Card component Card snapshot 1`] = `
<section
  className="card"
>
  Home
</section>
`;
